src/frontends/pytorch/src/op/instance_norm.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/batch_norm.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/gather.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/mvn.hpp"
+#include "openvino/op/range.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/squeeze.hpp"
+#include "openvino/op/tile.hpp"
+#include "openvino/op/unsqueeze.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_instance_norm(NodeContext& context) {
+    num_inputs_check(context, 8, 9);
+    auto input = context.get_input(0);
+    auto eps = static_cast<float>(context.const_input<double>(7));
+    auto input_shape = context.mark_node(std::make_shared<ov::op::v3::ShapeOf>(input));
+    auto rank_1d = context.mark_node(std::make_shared<ov::op::v3::ShapeOf>(input_shape));
```

Comment:
can you please align opset version for all nodes in the decomposition and use the latest opset?